app/utils/__init__.py

```diff
@@ -207,3 +207,12 @@ def get_defconfig_full(
             defconfig_full = defconfig_full_k
 
     return defconfig_full
+
+
+def make_path(dir_path):
+    if not os.path.isdir(dir_path):
+        try:
+            os.makedirs(dir_path)
+        except OSError as e:
+            if e.errno != errno.EEXIST:
```

Comment:
If a file exists with the name `dir_name`, then the directory won't be created and the exception won't be propagated either.  So it's failing silently in that case.  Is that right?

Reply:
This is an unexpected situation. I added a warning so we can spot it in the logs when it occurrs.